src/typegate/src/typegate/mod.ts

```diff
@@ -397,6 +397,12 @@ export class Typegate implements AsyncDisposable {
     await this.artifactStore.runArtifactGC();
   }
 
+  async forceRemoveAllTypegraphs() {
+    for (const engine of this.register.list()) {
+      await this.removeTypegraph(engine.name);
+    }
+  }
```

Comment:
We would want to run this before the register syncs.
So remove directly from the s3/redis/fs...

Reply:
Oh, just looked at the logs, that is a good point.